test/EFCore.SqlServer.FunctionalTests/JsonTypesSqlServerTest.cs

```diff
@@ -32,6 +32,33 @@ public override void Can_read_write_nullable_ulong_enum_JSON_values(object? valu
         base.Can_read_write_nullable_ulong_enum_JSON_values(value, json);
     }
 
+    public override void Can_read_write_collection_of_ulong_enum_JSON_values()
+        => Can_read_and_write_JSON_value(
+            Fixture.EntityType<PrimitiveTypeCollections>().GetProperty(nameof(PrimitiveTypeCollections.EnumU64)),
+            new List<EnumU64>
+            {
+                EnumU64.Min,
+                EnumU64.Max,
+                EnumU64.Default,
+                EnumU64.One,
+                (EnumU64)8
+            },
+            "{\"Prop\":[0,-1,0,1,8]}"); // Because ulong is converted to long on SQL Server
```

Comment:
DEATH TO THE BACKSLASHES! :skull: 

```suggestion
            """{"Prop":[0,-1,0,1,8]}"""); // Because ulong is converted to long on SQL Server
```